GroupProject/Main/clsMainSQL.cs

```diff
@@ -9,181 +9,167 @@ namespace GroupProject.Main
-                return "INSERT INTO LineItems(InvoiceNum, LineItemNum, ItemCode) Values(" + invoiceNum + ", " + lineItemNum + ", '" + itemCode + "');";
-            }
-            catch (Exception ex)
-            {
-                throw new Exception(MethodInfo.GetCurrentMethod().DeclaringType.Name + "." + MethodInfo.GetCurrentMethod().Name + " -> " + ex.Message);
-            }
+            string sSQL = "SELECT InvoiceNum, InvoiceDate, TotalCost " +
+                "FROM Invoices WHERE InvoiceNum =" + InvoiceNum;
+            return sSQL;
         }
 
         /// <summary>
-        /// Delete line item in LineItems table
+        /// This will Select and Return Items from the DB
         /// </summary>
-        /// <param name="invoiceNum">InvoiceNum</param>
         /// <returns></returns>
-        public string deleteLineItem(int invoiceNum)
+        public string SelectItems()
         {
-            try
-            {
-                return "DELETE From LineItems WHERE InvoiceNum = " + invoiceNum + ";";
-            }
-            catch (Exception ex)
-            {
-                throw new Exception(MethodInfo.GetCurrentMethod().DeclaringType.Name + "." + MethodInfo.GetCurrentMethod().Name + " -> " + ex.Message);
-            }
+            string sSQL = "SELECT ItemCode, ItemDesc, Cost " +
+                "FROM ItemDesc";
+            return sSQL;
         }
 
         /// <summary>
-        /// Delete a line item from an invoice
+        /// This will get all items
         /// </summary>
-        /// <param name="invoiceNum">InvoiceNum</param>
-        /// <param name="lineItemNum">LineItemNum</param>
         /// <returns></returns>
-        public string deleteLineItemsFromInvoice(int invoiceNum, int lineItemNum)
+        public string SelectAllItems()
         {
-            try
-            {
-                return "DELETE FROM ItemDesc WHERE InvoiceNum = " + invoiceNum + " AND LineItemNum =  " + lineItemNum + ";";
-
-            }
-            catch (Exception ex)
-            {
-                throw new Exception(MethodInfo.GetCurrentMethod().DeclaringType.Name + "." + MethodInfo.GetCurrentMethod().Name + " -> " + ex.Message);
-            }
+            return "SELECT * FROM ItemDesc";
         }
 
-
         /// <summary>
-        /// Get max invoice number
+        /// This will Select and Return The InvoiceNum and corresponding ItemNums
         /// </summary>
         /// <returns></returns>
-        public string getMaxInvoiceNum()
+        public string SelectLineItems(string InvoiceNum)
         {
-            try
-            {
-                return "SELECT MAX(InvoiceNum) from Invoices;";
-            }
-            catch (Exception ex)
-            {
-                throw new Exception(MethodInfo.GetCurrentMethod().DeclaringType.Name + "." + MethodInfo.GetCurrentMethod().Name + " -> " + ex.Message);
-            }
+            string sSQL = String.Format("SELECT InvoiceNum, LineItemNum, ItemCode " +
```

Comment:
Sorry one more bugging question on this, Has he asked us to use string formatter or is interpolation fine as well?

Reply:
I don't think he asked for anything specific to format the strings, just as long as they work correctly